tests/extension/oneapi_local_memory/local_memory.cpp

```diff
@@ -0,0 +1,256 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2024 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../common/common.h"
+
+namespace local_memory::tests {
+
+#ifdef SYCL_EXT_ONEAPI_LOCAL_MEMORY
+
+struct Point3D {
+  int x;
+  int y;
+  int z;
+  Point3D() : x{0}, y{0}, z{0} {}
+  Point3D(int x) : x{x}, y{0}, z{0} {}
+  Point3D(int x, int y) : x{x}, y{y}, z{0} {}
+  Point3D(int x, int y, int z) : x{x}, y{y}, z{z} {}
```

Comment:
```suggestion
  Point3D(int x = 0, int y = 0, int z = 0) : x{x}, y{y}, z{z} {}
```

Reply:
Otherwise, why 0 everywhere as it might hide some bugs? What about some negative and positive prime numbers?